README.md

```diff
@@ -18,6 +18,15 @@ schemas are used for direct to parquet output; some examples of Parquet-MR
 schemas can be found here:
 [Parquet Schema Examples](https://mozilla-services.github.io/lua_sandbox_extensions/parquet/io_modules/lpeg/parquet.html)
 
+## Adding a new schema
+
+- Create the JSON Schema in the `templates` directory first. Make use of common schema components from the `templates/include` directory where possible, including things like the telemetry `environment`, `clientId`, `application` block, or UUID patterns. The filename should be `templates/<namespace>/<doctype>/<doctype>.<version>.schema.json`.
+- If the data will be saved in parquet form, also add a Parquet-MR schema at `templates/<namespace>/<doctype>/<doctype>.<version>.parquetmr.txt`.
+- Build the rendered schemas using the instructions below, and check those artifacts (in the `schemas` directory) in to the git repo as well. See the rationale for this in the "Notes" section below.
+- Add one or more example JSON documents to the `validation` directory.
+- Run the tests via Docker using the instructions below.
```

Comment:
You don't have to run the tests via docker. The set of people for which this is relevant already know how to run the tests without it, though (and it's doc'd later as "optional").